wcfsetup/install/files/style/ui/userCard.scss

```
@@ -0,0 +1,130 @@
+.userCardList {
+    display: grid;
+    gap: 20px;
+    grid-auto-rows: minmax(200px, auto);
+    padding: 20px 0;
+
+    @include screen-xl {
+        grid-template-columns: repeat(3, 1fr);
+    }
+
+    @include screen-lg-only {
+        grid-template-columns: repeat(2, 1fr);
+    }
+
+    @include screen-md {
+        grid-template-columns: repeat(3, 1fr);
+    }
+
+    @include screen-sm {
+        grid-template-columns: repeat(2, 1fr);
+    }
+}
+
+.userCard {
+    background-color: var(--wcfContentBackground);
+    border-radius: var(--wcfBorderRadius);
+    box-shadow: var(--wcfBoxShadowCard);
+    display: flex;
+    flex-direction: column;
+    position: relative;
+}
+
+.userCard__header {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+}
+
+.userCard__header__background {
+    height: 100px;
+    overflow: hidden;
+    border-top-right-radius: var(--wcfBorderRadius);
+    border-top-left-radius: var(--wcfBorderRadius);
+}
+
+.userCard__header__background__image {
+    height: 100%;
+    object-fit: cover;
+    max-width: 100%;
+}
+
+.userCard__header__avatar {
+    position: relative;
+    border-radius: 50%;
+    border: 5px solid var(--wcfContentBackground);
+    margin: auto;
+    margin-top: -37px;
+}
+
+.userCard__onlineIndicator {
+    background-color: rgba(0, 153, 0, 1);
+    border: 1px solid var(--wcfContentBackground);
+    border-radius: 50%;
+    bottom: 0;
+    height: 12px;
+    position: absolute;
+    right: 6px;
+    width: 12px;
+}
+
+.userCard__footer {
+    border-top: 1px solid var(--wcfContentBorderInner);
+    padding: 10px;
+}
+
+.userCard__content {
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    flex: 1 auto;
+    row-gap: 10px;
+    padding: 20px;
+}
+
+.userCard__username {
+    @include wcfFontHeadline;
+    @include wcfFontBold;
+
+    a {
+        color: inherit;
+    }
+}
+
+.userCard__footer__stats {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 10px;
+}
+
+.userCard__footer__statsItem {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+}
+
+.userCard__footer__statsItem:nth-child(n + 4) {
+    display: none;
+}
+
+.userCard__footer__statsItem__key {
+    @include wcfFontSmall;
+    color: var(--wcfContentDimmedText);
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+
+.userCard__buttons {
+    background-color: var(--wcfContentContainerBackground);
+    border-radius: 30px;
+    display: flex;
+    justify-content: center;
+    gap: 5px;
+    margin: 0 auto;
+    padding: 5px 10px;
+
+    a {
```

```suggestion
    .userCard__buttons {
```

I’m not entirely sure if it works, you’ll have to try it out.